### ISSUE TITLE
Maven build for roda-core-jar always fails because of some docker stuff...

### ISSUE THREAD

**Issue by rui-castro** [keeps/roda#554]
## Description

Building roda-core-jar with the following command line (as @hsilva-keep) [suggested](https://github.com/keeps/roda/issues/547#issuecomment-247926071) always fails after creating the roda-core jar files.

```bash
mvn -Proda-core-jar,core clean package -Dmaven.test.skip=true
```

The logs of the failling part bellow.

```
...
[INFO] Building image roda-core
Sep 21, 2016 10:54:24 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: Permission denied
Sep 21, 2016 10:54:24 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->unix://localhost:80
Sep 21, 2016 10:54:24 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: Permission denied
Sep 21, 2016 10:54:24 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->unix://localhost:80
Sep 21, 2016 10:54:24 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: Permission denied
Sep 21, 2016 10:54:24 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {}->unix://localhost:80
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] RODA - Repositório de Objetos Digitais Autênticos .. SUCCESS [  6.108 s]
[INFO] roda-common ........................................ SUCCESS [  0.118 s]
[INFO] roda-common-data - Common objects/classes used by RODA SUCCESS [ 24.257 s]
[INFO] roda-common-utils - Utility classes used by RODA ... SUCCESS [ 10.970 s]
[INFO] roda-common-certification .......................... SUCCESS [ 15.629 s]
[INFO] roda-core .......................................... FAILURE [01:37 min]
[INFO] roda-core-tests .................................... SKIPPED
[INFO] roda-plugin-common ................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:35 min
[INFO] Finished at: 2016-09-21T10:54:24+01:00
[INFO] Final Memory: 109M/836M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.4.10:build (default) on project roda-core: Exception caught: java.util.concurrent.ExecutionException: com.spotify.docker.client.shaded.javax.ws.rs.ProcessingException: java.io.IOException: Permission denied -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :roda-core
```

**Comment by hsilva-keep** [comment]
@rui-castro It fails because docker is not configured properly in your machine. 
Nevertheless, a profile specific for creating a docker image with roda-core artifact will be created & therefore this will only happen when one explicitly invokes it. Then, you will have to deal with your machine configuration.

**Comment by rui-castro** [comment]
@hsilva-keep :+1:

**Comment by hsilva-keep** [comment]
Solved since b069d99822fa7ec2b6d8930694021b36e272bcea.
Generate only roda core jar:
`$> mvn -Proda-core-jar,core clean package -Dmaven.test.skip=true`
Generate both roda core jar & docker image:
`$> mvn -Proda-core-jar-docker,core clean package -Dmaven.test.skip=true`
